etna/blog/models.py

```diff
@@ -0,0 +1,130 @@
+from django.utils.functional import cached_property
+
+from wagtail.admin.panels import FieldPanel
+from wagtail.api import APIField
+from wagtail.fields import StreamField
+
+from etna.core.models import (
+    BasePageWithRequiredIntro,
+    ContentWarningMixin,
+    HeroImageMixin,
+)
+from etna.core.serializers import DefaultPageSerializer
+from etna.people.models import AuthorPageMixin
+
+from .blocks import BlogPostPageStreamBlock
+
+
+class BlogIndexPage(HeroImageMixin, BasePageWithRequiredIntro):
+    """Blog index page
+
+    This is the parent page for all blog posts. It is used to
+    display a list of blog posts, and blog pages.
+    """
+
+    subpage_types = ["blog.BlogPostPage", "blog.BlogPage"]  # TBC
+
+    parent_page_types = ["home.HomePage"]  # TBC
+
+    content_panels = (
+        BasePageWithRequiredIntro.content_panels + HeroImageMixin.content_panels
+    )
+
+    promote_panels = BasePageWithRequiredIntro.promote_panels
+
+    @cached_property
+    def blog_pages(self):
+        return (
+            BlogPage.objects.all()
+            .order_by("-first_published_at")
+            .live()
+            .public()
+            .specific()
+        )
+
+    api_fields = BasePageWithRequiredIntro.api_fields + [
+        APIField("blog_pages", serializer=DefaultPageSerializer(many=True))
+    ]
+
+
+class BlogPage(HeroImageMixin, BasePageWithRequiredIntro):
+    """Blog page
+
+    This is the parent page for blog posts
+    It is used to display a list of the blog posts
+    that are children of this page, as well as other
+    blogs within this blog.
+    """
+
+    subpage_types = ["blog.BlogPostPage", "blog.BlogPage"]
+
```

Comment:
```suggestion
parent_page_types = ["blog.BlogPage", "BlogIndexPage"]

```

...else these could be added anywhere on the site.

Reply:
Did we not want to make these appear anywhere on the site? I thought we were going to make it so that we can have them under Explore the Collection, Exec team pages, etc?

Reply:
Oops, yes - I am an idiot and I think I was tangled up in the model names!